### ISSUE TITLE
Question: is this still the best way to get BCM94360 working?

### ISSUE THREAD

**Issue by aforty** [billabongbruno/macOS-Sonoma-Broadcom-Wifi#1]
Hi. I Just came across this repo and I was wondering if this is indeed still the best way to get BCM94360 working in Sonoma or has the installation method evolved/changed since you wrote this (for beta2). Any resources you could point me to would be really appreciated. Thanks for writing this!

**Comment by billabongbruno** [comment]
> Hi. I Just came across this repo and I was wondering if this is indeed still the best way to get BCM94360 working in Sonoma or has the installation method evolved/changed since you wrote this (for beta2). Any resources you could point me to would be really appreciated. Thanks for writing this!

Hey there.

I have been checking regularly (I did so 2 days ago)m and the only thing that seems to have been updated is OCLP itself, which is now on version 1.0.1, found [here](https://github.com/dortania/OpenCore-Legacy-Patcher/releases/tag/1.0.1) and fully working with the official Sonoma release.

Other than that, no changes I can find and hope for (like some combination of kexts without root patching).

Hopefully they will come.

If anything changes, I'll edit this repo accordingly.


You're welcome and good luck.
